src/CodingContract/contracts/TotalPrimesInRange.ts

```diff
@@ -1,14 +1,15 @@
 import { CodingContractName } from "@enums";
-import { removeBracketsFromArrayString, type CodingContractTypes } from "../ContractTypes";
+import { CodingContractTypes } from "../ContractTypes";
 import { getRandomIntInclusive } from "../../utils/helpers/getRandomIntInclusive";
 
 export const totalPrimesInRange: Pick<CodingContractTypes, CodingContractName.TotalPrimesInRange> = {
   [CodingContractName.TotalPrimesInRange]: {
     desc: (data: number[]): string => {
       return [
         `You are given two random non-negative integers: ${data}.\n`,
```

Comment:
I should have mentioned this earlier: We should be a bit more explicit about the limits of the range, so people can better plan for what size of data they'll need to handle.
```suggestion
        `You are given two random non-negative integers: ${data}. The first will be in the range [0, 5000000] and the second will be at most 1000000 greater.\n`,
```